addons/barfiller/data/settings.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.1" ?>
```

Comment:
Do not provide a settings file, it will be automatically generated. The data folder should generally stay empty.